src/components/NSLookupTool/NSLookupTool.tsx

```diff
@@ -7,50 +7,54 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
```

Comment:
You do not need to import UserGuide because it is deprecated. You need to import checkAllCommandsAvailability, InstallationModal and RenderComponent. 

```suggestion
import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
import InstallationModal from "../InstallationModal/InstallationModal";
import { RenderComponent } from "../UserGuide/UserGuide";
```